### DISCUSSION TITLE
(suivi achats) est-ce qu'on peut avoir plusieurs produits dans un achat ?

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#995] [Question / réponse] [answered]
Je pense maintenant c'est pas super clair si l'utilisateur.rice doit ajouter une ligne pour chaque produit acheté ou chaque facture.

Je pense pour nos calculs, en particulière les totaux bio, durable, HVE etc qu'on veut pré-remplir dans le diagnostic, on veut avoir une ligne pour chaque produit (en pensant que c'est possible que je vais acheter des œufs label rouge et du lait bio en même temps du même vendeur, mais il faut qu'on ne calcule pas les œufs dans le total bio etc)

**Comment by alemangui** [comment, marked as answer]
C'est ça, l'unité c'est l'achat d'un produit. Ça permet de spécifier les labels et les caractéristiques (sinon il faudrait re-spécifier pour chaque produit). Donc une même facture peut être téléversée pour plusieurs lignes, mais ce n'est pas grave car ce champ est là seulement pour faciliter la vie à l'utilistaeur, on ne fait aucun traitement là-dessus.

**Reply by hfroot** [reply]
Est-ce que c'est assez claire niveau UX ? Peut-être c'est une question pour l'équipe bizdev.
- Par exemple "Ajouter une nouvelle ligne" -> "Ajouter un nouveau achat"; et ajouter un peu de texte "Si vous avez plusieurs achats/produits listé sur votre facture, ajoutez un nouveau achat pour chacun" (mais p-e il y a qqch autre qu'on peut faire sans écrire des explications en dernier recours...)

Est-ce que c'est commun d'avoir plusieurs produits sur un facture ?
- Peut-être si oui on peut avoir un bouton "Valider et ajouter un nouveau pour ce facture" plutôt que le bouton que j'ai ajouté, qui va garder le fichier, le date, la cantine, le nom du fournisseur et juste laisse les labels et le catégorie vides...

**Reply by hfroot** [reply]
En plus, p-e on a besoin d'un champ "Nom produit" ou qqch

**Reply by alemangui** [reply]
D'accord avec toi, et le champ "produit" est facile à mettre en place. À discuter avec Jenn aussi lors du planning.

**Comment by hfroot** [comment]
#1067
